### ISSUE TITLE
Missing the modBehavior documentation and explanation

### ISSUE THREAD

**Issue by garubi** [DecentSamples/DecentSampler-Developers-Guide#2]
In the documentation thare's no mention of the `modBehavior ` attribute.

It should be sufficient to copy-paste the explanation found here:
https://www.decentsamples.com/2022/08/19/how-to-add-lfos-and-extra-envelopes-to-your-decent-sampler-instruments/ (see paragraph titled "The new modBehavior parameter for bindings".

I can submit a pull request but I would like to have some direction on where to put the new section: in the Modulaters page (docs/source/the-modulators-element.md) or in the Bindings appendix (docs/source/appendix-b-the-binding-element.md)?

**Comment by dhilowitz** [comment]
You're right! It's been on my to-do list for ages. I'll try to get to it once my next video is done. :)
